install.sh

```diff
@@ -115,6 +115,11 @@ if [[ ! -f "$SCRIPT" || ! -f "$WRAPPER" || ! -f "$CONFIGFILE" ]]; then
   exit 1
 fi
 
+## if $APPDIR is missing then create it
+if [[ ! -d "$APPDIR" ]]; then
```

Comment:
Don't check if it's not there, just do `mkdir -vp "$APPDIR"`. Put it after `mkdir -vp "$CONFIGDIR"`.